src/BenchmarkDotNet/Toolchains/MonoWasm/WasmGenerator.cs

```diff
@@ -96,14 +96,7 @@ protected void GenerateProjectInterpreter(BuildPartition buildPartition, Artifac
 
         protected override string GetBinariesDirectoryPath(string buildArtifactsDirectoryPath, string configuration)
         {
-            if (Aot)
```

Comment:
I don't understand this change and I would like to be sure that we are not breaking anything. Have you tested both AOT and non-AOT toolchains?

Reply:
Basically, before wasm-intepreter mode was using a  simplified custom app builder I built into BDN, because that was the best thing that could be done at the time. 

Wasm-aot used all the app bundling stuff from the the runtime source directory, which put things in a different place than the one I coded into BDN.

With this change, both wasm-interpreter and wasm-aot will use the app builder from the runtime source dir, so the files will end up in the same place.

Reply:
I've tested both AOT and interpreter configurations.